### PR TITLE
repositories.xml

### PR DESCRIPTION
**Pull request by sheerluck**
for [layman](https://wiki.gentoo.org/wiki/Layman#Adding_custom_repositories)

### CODE REVIEW

File: repositories.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE repositories SYSTEM "/dtd/repositories.dtd">
+<repositories xmlns="" version="1.0">
+  <repo priority="50" quality="experimental" status="unofficial">
+    <name><![CDATA[ml-overlay]]></name>
```
Comment (aballier) [inline repositories.xml]: why CDATA here and there ?

Comment (sheerluck) [reply repositories.xml]: Do not know, I just copied repositories.xml from flatpak-overlay :)

Comment (aballier) [reply repositories.xml]: then please remove it: https://api.gentoo.org/overlays/repositories.xml
does not use that

**Comment by sheerluck** [comment]
I removed CDATA

**Pull request merged**